modules/olm-overriding-operator-pod-affinity.adoc

```diff
@@ -0,0 +1,307 @@
+
+The following examples describe situations where you might want to schedule an Operator pod to a specific node or set of nodes:
+
+* If an Operator requires a particular architecture, such as `amd64` or `arm64`
+* If an Operator requires a particular operating system, such as Linux or Windows
+* If you want Operators that work together scheduled on the same host or on hosts located on the same rack
+* If you want Operators dispersed throughout the infrastructure to avoid downtime due to network or hardware issues
+
+ifdef::olm[]
+You can control where an Operator pod is installed by adding node affinity, pod affinity, or pod anti-affinity constraints to the Operator's `Subscription` object. Node affinity is a set of rules used by the scheduler to determine where a pod can be placed. Pod affinity enables you to ensure that pods are scheduled to the same node. Pod anti-affinity allows you to prevent a pod from being scheduled on a node. 
+endif::olm[]
+
+ifdef::pod[]
+You can control where an Operator pod is installed by adding a pod affinity or anti-affinity to the Operator's `Subscription` object. 
+endif::pod[]
+
+ifdef::node[]
+You can control where an Operator pod is installed by adding a node affinity constraints to the Operator's `Subscription` object.
+endif::node[]
+
+ifdef::olm[]
+The following examples show how to use node affinity or pod anti-affinity to install an instance of the Custom Metrics Autoscaler Operator to a specific node in the cluster:
+endif::olm[]
+ifdef::node[]
+The following examples show how to use node affinity to install an instance of the Custom Metrics Autoscaler Operator to a specific node in the cluster:
+endif::node[]
+
+ifndef::pod[]
+.Node affinity example that places the Operator pod on a specific node
+[source,terminal]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  creationTimestamp: '2022-12-21T14:54:47Z'
+  generation: 6
+  labels:
+    operators.coreos.com/openshift-custom-metrics-autoscaler-operator.openshift-keda: ''
+  name: openshift-custom-metrics-autoscaler-operator
+  namespace: openshift-keda
+  resourceVersion: '77887'
+  uid: 9b38d0eb-9c57-4cb6-9905-1c65c21f08ef
+spec:
+  name: my-package
+  source: my-operators
+  sourceNamespace: operator-registries
+  config:
+    affinity:
+      nodeAffinity: <1>
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+          - matchExpressions:
+            - key: kubernetes.io/hostname
+              operator: In
+              values:
+              - ip-10-0-163-94.us-west-2.compute.internal
+ ...
+----
+<1> A node affinity that requires the Operator's pod to be scheduled on a node named `ip-10-0-163-94.us-west-2.compute.internal`.
+
+.Node affinity example that places the Operator pod on a node with a specific architecture
```

Comment:
This is super nit-picky, but I would prefer to see us use the term "platform" instead of architecture.

linux/amd64 != windows/amd64, but they're they same architecture.